pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -239,6 +243,7 @@ func (c *collector) collectPodResUsed() {
 			}
 			continue
```

Comment:
Once failed to read one cgroup file, I think we should not skip to read left collections.

Reply:
Here both the pod's cpu usage and memory usage are considered, which often fail at the same time (e.g. pod is restarting, or the cgroup path is expired).